debian/changelog

```diff
@@ -1,3 +1,9 @@
+packetfence (9.1.9-1) unstable; urgency=low
```

Comment:
Is it possible in the ci to use dch -v ?

Reply:
That's already the case, see https://gitlab.com/Orange-OpenSource/gitlab-buildpkg-tools/blob/master/gitlab-buildpkg-tools/usr/bin/ci-build-pkg#L109-112.

In fact, `gitlab-buildpkg-tools` will update `debian/changelog` and SPEC files (RPM) in the pipeline before running the build so you got packages with following names:
- DEB: `packetfence_9.1.9+20191015120322+89377110+feature~ci~build~pkg~rpm+stretch+olab1_all.deb`
- RPM: `packetfence-9.1.9-20191015120322.89377110.feature~ci~build~pkg~rpm.olab.el7.x86_64.rpm`

Revision is not really used, we rely on date and pipeline ID.

So each time you make a new commit on devel, it will generate new packages. It permits to have a Continuous Integration system.